### PR TITLE
Sysroot fix 10.2

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Update from master and sysroot fix

### CODE REVIEW

File: .ci_support/linux_ppc64le_.yaml [hunk 1]
```diff
@@ -1,21 +0,0 @@
-c_compiler:
```
Comment (jakirkham) [inline .ci_support/linux_ppc64le_.yaml:1]: Was this suppose to be removed?

Comment (scopatz) [reply .ci_support/linux_ppc64le_.yaml:1]: Oh probably not. Let me rerender.

Comment (scopatz) [reply .ci_support/linux_ppc64le_.yaml]: done!

**Comment by jakirkham** [comment]
Looks good. Thanks! 😄

**Comment by jakirkham** [comment]
Did we want to do this for CUDA 11.0 as well? I guess it doesn't technically have the issue we had here, but we can align the recipe at least.

**Pull request merged**